libdevice/sanitizer/tsan_rtl.cpp

```diff
@@ -115,6 +115,11 @@ inline __SYCL_GLOBAL__ RawShadow *MemToShadow_PVC(uptr addr, uint32_t as) {
 inline __SYCL_GLOBAL__ RawShadow *MemToShadow(uptr addr, uint32_t as) {
   __SYCL_GLOBAL__ RawShadow *shadow_ptr = nullptr;
 
+#if defined(__LIBDEVICE_CPU__)
```

Comment:
Please also apply such changes to GetCurrentSid, __tsan_cleanup_private, __tsan_group_barrier